### PR TITLE
V2

### PR DESCRIPTION
**Pull request by winkerVSbecks**

### CODE REVIEW

File: src/components/screens/DocsScreen/VersionSelector.tsx [hunk 1]
```diff
@@ -1,113 +1,59 @@
...
-  line-height: 18px;
-  color: ${color.mediumdark};
-  padding: 7px 15px;
-  border-bottom: 1px solid ${color.border};
-  ${(props) => props.withTopBorder && `border-top: 1px solid ${color.border};`}
-`;
+interface PreRelease {
+  version: number;
+  string: string;
+  label: 'alpha' | 'beta' | 'rc';
```
Comment (kylegach) [inline src/components/screens/DocsScreen/VersionSelector.tsx:19]: `7.0` also had a `'future'` label initially, but I'm not sure if that was just a one-time thing. Probably can't hurt to include it.

File: src/components/screens/DocsScreen/VersionSelector.tsx [hunk 2]
```diff
@@ -1,113 +1,59 @@
...
-        <VersionLink isButton primary withArrow>
-          {stylizeVersion(
-            [...versions.stable, ...versions.preRelease].find(({ version: v }) => v === version)
-          )}
-        </VersionLink>
-      </WithTooltip>
-    </Wrapper>
+  const versionOptions = [
+    {
+      label: 'stable',
```
Comment (kylegach) [inline src/components/screens/DocsScreen/VersionSelector.tsx:45]: I think these `label`s should be capitalized.

Comment (winkerVSbecks) [reply src/components/screens/DocsScreen/VersionSelector.tsx:45]: Like 'Stable' instead of `stable`?

Comment (kylegach) [reply src/components/screens/DocsScreen/VersionSelector.tsx:45]: Yep. Same for "Pre-release".

Comment (kylegach) [reply src/components/screens/DocsScreen/VersionSelector.tsx:45]: Ha. I left these comments before looking at the snapshots/Storybook. I see now that the visual treatment renders this moot. 🤦‍♂️ Sorry for the noise!

I'll take the opportunity to say that both FrameworkSelector and VersionSelector seem sound and look great to me!

**Pull request merged**